apps/frontend/src/libs/components/breadcrumbs/breadcrumbs-item/breadcrumbs-item.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon } from "~/libs/components/components.js";
+import { type BreadcrumbNavigationItem } from "~/libs/components/types/breadcrumb-navigation-item.type.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	isLastItem: boolean;
+} & BreadcrumbNavigationItem;
+
+const BreadcrumbItem = ({
+	href,
+	isLastItem,
+	label,
+}: Properties): JSX.Element => {
+	return (
+		<li className={styles["breadcrumb-item"]}>
+			{isLastItem ? (
```

Comment:
to display a link, we need to check if link passed, but to display the separator we need use isLastItem
```suggestion
			{hasHref ? (
```